consensus/pacemaker.go

```diff
@@ -36,6 +37,8 @@ var _ PacemakerDebug = &paceMaker{}
 type paceMaker struct {
 	bus modules.Bus
 
+	m sync.RWMutex
```

Comment:
Can you add a comment about why this is needed and when it should/shouldn't be used.

Reply:
Removed as not used anymore in https://github.com/pokt-network/pocket/pull/198/commits/37a0bd8be657b48c365b20c424513121679f0e49